src/gobprepare/utils/postgres.py

```diff
@@ -39,3 +39,44 @@ def create_table_columnar_query(postgres_datastore: PostgresDatastore, tablename
     if _is_columnar_supported(postgres_datastore):
         return f"CREATE TABLE {tablename} ({columndefs}) USING columnar"
     return f"CREATE TABLE {tablename} ({columndefs})"
+
+def check_table_existence_query(schema: str, table_name: str) -> str:
```

Comment:
Why not return a boolean when table exists yes/no

Reply:
Deze functie geeft de (existence) query terug. Niet de resultaat van de uitvoering.